src/main/java/com/google/cloud/teleport/v2/bigquery-gcs/src/test/java/com/google/cloud/teleport/v2/bigquerygcs/BigQueryToParquetTest.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.teleport.v2.bigquerygcs;
```

Comment:
use com.google.cloud.teleport.v2.templates for the package.

Reply:
Updated package name.